src/main/java/net/minecraftforge/java_provisioner/Disco.java

```diff
@@ -636,43 +692,55 @@ private static Arch getCurrent() {
         }
     }
 
+    // TODO [Provisioner][Distro] Reconsider the order of this entire list
+    //      Prefer TCK certified JDKs (and also test Microsoft)
     public enum Distro implements Comparable<Distro> {
-        // These are well know/recommended distros
+        // These are well-known/recommended distros
```

Comment:
So, discussing this on discord, just reiterating it here. I don't particularly care.
But `MS > OpenJDK > Temurin > * > JDK > OpenJ9` seems to be the consensus on the order.